packages/core/src/external/fhir/consolidated/consolidated.ts

```diff
@@ -106,7 +102,7 @@ export async function getConsolidatedFhirBundle({
     });
   }
 
-  let filtered = filterByDocumentIds(success, documentIds, log);
```

Comment:
Only used by `POST /internal/docs/re-convert`, so moved to `packages/api/src/command/medical/patient/consolidated-delete.ts`